www/source/demo/process-supervisor/steps/10.html.slim

```diff
@@ -35,11 +35,15 @@ classes: demo_step
         = code(:shell) do
           | $ hab studio enter
         = code(:shell) do
-          | [1][default:/src:0]# hab start yourorigin/sample-node-app --strategy at-once
+          |
+            [1][default:/src:0]# hab sup run
```

Comment:
If this is being done in a studio (line 36), then there's already a supervisor running in the background.